src/dsp_tools/xmllib/__init__.py

```diff
@@ -1,4 +1,8 @@
-from .helpers import create_label_to_name_list_node_mapping as create_label_to_name_list_node_mapping
+from .helpers import create_label_to_name_list_node_mapping
```

Comment:
is there a reason why you imported everything with an alias?

Reply:
I thought aliasing would be necessary for the linters, but if it works without aliases, I don't care. You can remove the aliases everywhere, to be consistent.